app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/BuildingsFragment.java

```diff
@@ -89,7 +89,6 @@ public void onClick(View v) {
                 apiService.postBuilding(preferences.getString(USER_ACCESS_TOKEN, ""), "farm").enqueue(new Callback<Building>() {
                     @Override
                     public void onResponse(Call<Building> call, Response<Building> response) {
-                        apiService.addBuildingToList(response.body());
                         refresh();
```

Comment:
@rekagiday when just adding one building we can add it directly without refreshing, since its in the response